### PR TITLE
hwdb: Add accelerometer orientation quirk for the Teclast X98 Plus I

### PR DESCRIPTION
**Pull request by javsanpar**
Add accelerometer orientation quirk for the Teclast X98 Plus I (A5C6) tablet.

### CODE REVIEW

**Comment by keszybz** [comment]
There's a trailing space in the empty line you added. Please remove.

File: hwdb/60-sensor.hwdb [hunk 1]
```diff
@@ -388,6 +388,10 @@ sensor:modalias:acpi:KIOX000A*:dmi:*:svnTECLAST:pnX80Pro:*
 
 sensor:modalias:acpi:KIOX000A*:dmi:*:svnTECLAST:pnX98PlusII:*
  ACCEL_MOUNT_MATRIX=1, 0, 0; 0, -1, 0; 0, 0, 1
+ 
+# Teclast X98 Plus I (A5C6), generic DMI strings, match entire dmi modalias inc. bios-date
+sensor:modalias:acpi:KIOX000A*:dmi:bvnAmericanMegatrendsInc.:bvr5.011:bd11/03/2015:svnTobefilledbyO.E.M.:pnTobefilledbyO.E.M.:pvrTobefilledbyO.E.M.:rvnAMICorporation:rnCherryTrailCR:rvrTobefilledbyO.E.M.:cvnToBeFilledByO.E.M.:ct3:cvrToBeFilledByO.E.M.:
```
Comment (yuwata) [inline hwdb/60-sensor.hwdb]: This seems too specific. Why do you include the bios vendor or bios date?

Comment (javsanpar) [reply hwdb/60-sensor.hwdb]: I have based myself on this commit: f253cee808d435975043ca903c382b0b942e600c

The tablet added to that commit seems to have a string as generic as mine.

**Comment by keszybz** [comment]
Yeah, I guess that this is another case where the bios date is the only real bit of information.

**Comment by keszybz** [comment]
semaphoreci already runs the parsing tests, the rest of CI does not provide additional tests, so we can merge this.

**Pull request merged**